idseq_dag/util/command.py

```diff
@@ -186,25 +189,35 @@ def subprocess_scope(*args, **kwargs):
     return wrapper
 
 
-def retry(operation, randgen=random.Random().random):
+def retry(operation, MAX_TRIES=3):
```

Comment:
A bit of context on this.   The version we had here was fine for multiple threads, which is really mostly what we do in IDseq-DAG, but in my use on another project I discovered it wasn't great when using multiple processes.  We don't use multiple processes that aggressively in IDseq, and the only place where we do is lzw, which to my knowledge does not do retries.  However, since there is now a more robust version, that works better also for multiple processes --- and is also smarter to back off longer for longer requests, which is the right behavior if the problem is an overloaded service --- why not just upgrade.